src/components/PlioQuestion.vue

```diff
@@ -239,23 +294,11 @@ li {
   border-radius: 5px;
 }
 
-.answer_option-correctAnswer {
-  text-align: left;
-  padding: 2px;
-  margin: 5px;
-  font-size: 1.3rem;
-  margin-right: 57px;
-  border-radius: 5px;
+.correctAnswer {
   background-color:lightgreen
 }
```

Comment:
.correctAnswer is added to the classlist of the required element -> it means that just the property we need is added